src/targets/browser/browserPathResolver.ts

```diff
@@ -2,44 +2,99 @@
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
 
+import { inject, injectable } from 'inversify';
 import * as path from 'path';
-import * as utils from '../../common/urlUtils';
-import { ISourcePathResolverOptions, SourcePathResolverBase } from '../sourcePathResolver';
-import { IUrlResolution } from '../../common/sourcePathResolver';
+import { URL } from 'url';
+import { IVueFileMapper, VueHandling } from '../../adapter/vueFileMapper';
+import { IFsUtils } from '../../common/fsUtils';
+import { ILogger, LogTag } from '../../common/logging';
 import {
-  properResolve,
   fixDriveLetterAndSlashes,
-  properRelative,
   isSubdirectoryOf,
+  properRelative,
+  properResolve,
 } from '../../common/pathUtils';
-import { PathMapping } from '../../configuration';
-import { URL } from 'url';
 import { SourceMap } from '../../common/sourceMaps/sourceMap';
 import {
-  getFullSourceEntry,
   defaultPathMappingResolver,
   getComputedSourceRoot,
+  getFullSourceEntry,
 } from '../../common/sourceMaps/sourceMapResolutionUtils';
-import { injectable, inject } from 'inversify';
-import { IVueFileMapper, VueHandling } from '../../adapter/vueFileMapper';
-import { ILogger } from '../../common/logging';
-import { IFsUtils } from '../../common/fsUtils';
+import { IUrlResolution } from '../../common/sourcePathResolver';
+import * as utils from '../../common/urlUtils';
+import { PathMapping } from '../../configuration';
+import { ISourcePathResolverOptions, SourcePathResolverBase } from '../sourcePathResolver';
 
 interface IOptions extends ISourcePathResolverOptions {
   baseUrl?: string;
   pathMapping: PathMapping;
   clientID: string | undefined;
+  isBlazor: boolean;
+  remoteFilePrefix: string | undefined;
 }
 
 @injectable()
 export class BrowserSourcePathResolver extends SourcePathResolverBase<IOptions> {
+  private readonly blazorInCodespacesRegexp: RegExp;
```

Comment:
Please make a new `BlazorSourcePathResolver` that extends the `BrowserSourcePathResolver` and then inject it here

https://github.com/microsoft/vscode-js-debug/blob/9b24a1420f7ce6b4a587e2386de3646ab4834101/src/targets/sourcePathResolverFactory.ts#L26

To avoid adding a ton of blazor special-casing into the generic browser path resolver

Reply:
Done